release/pkg/manager/calico/manager.go

```diff
@@ -80,6 +83,9 @@ var (
 		"csi",
 		"ctl",
 		"dikastes",
+		"envoy-gateway",
+		"envoy-proxy",
+		"envoy-ratelimit",
```

Comment:
pls update the components in `release/internal/pinnedversion/templates/calico-versions.yaml.gotmpl` with the images

Reply:
Thanks @radTuti for catching all those!